serializers/github_cache_entry.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class Serializers::GithubCacheEntry < Serializers::Base
+  def self.serialize_internal(entry, options = {})
+    {
+      id: entry.ubid,
+      key: entry.key,
+      repository_name: entry.repository.name,
+      scope: entry.scope,
+      created_at: entry.created_at.strftime("%Y-%m-%d %H:%M UTC"),
+      created_at_human: humanize_time(entry.created_at),
+      last_accessed_at: entry.last_accessed_at&.strftime("%Y-%m-%d %H:%M UTC"),
+      last_accessed_at_human: humanize_time(entry.last_accessed_at),
+      size: entry.size,
+      size_human: humanize_size(entry.size)
+    }
+  end
+
+  def self.humanize_size(bytes)
+    return nil if bytes.nil? || bytes.zero?
+    units = %w[B KB MB GB]
+    exp = (Math.log(bytes) / Math.log(1024)).to_i
+    exp = [exp, units.size - 1].min
+
+    "%.2f %s" % [bytes.to_f / 1024**exp, units[exp]]
```

Comment:
nit: do we need the .2f granularity? let's just make it easy to the eye

Reply:
Do you prefer just integer or .1f granularity? I believe there's a difference between `3MB` and `3.8MB`.

Reply:
.1f sounds good.